.github/workflows/ci.yml

```diff
@@ -89,3 +89,12 @@ jobs:
           else
             stack build --ghc-options="-Werror" --extra-include-dirs=$HOME/.local/include --extra-lib-dirs=$HOME/.local/lib
           fi
+          stack install --ghc-options="-Werror" --extra-include-dirs=$HOME/.local/include --extra-lib-dirs=$HOME/.local/lib
```

Comment:
is this necessary?

Reply:
I'm not 100% sure `stack build` will generate the binary (e.g. linking)